includes/class-suggestion-engine.php

```
@@ -0,0 +1,347 @@
+		private static function make( string $metric, float $value, string $unit, array $thresholds, string $fix_action, string $description ): array {
+			if ( $value <= $thresholds['good'] ) {
+				$status     = 'good';
+				$fix_action = 'no_action_required';
+			} elseif ( $value <= $thresholds['poor'] ) {
+				$status = 'needs_improvement';
+			} else {
+				$status = 'poor';
+			}
+
+			return self::build( $metric, $value, $unit, $status, $description, $fix_action );
+		}
```

PHPMD (2.15.0)

[error] The parameter $fix_action is not named in camelCase. (undefined)

(CamelCaseParameterName)